docs/_documentations/mdt-che-odo-support.md

```diff
@@ -24,21 +24,19 @@ The extension to Codewind provides support for [OpenShift Do (odo)](https://gith
 
 ## Setting up extension
 
-### Adding rules to support the extension
-The extension needs additional rules for accessing OpenShift resources. Use the following commands to clone the [codewind-odo-extension](https://github.com/eclipse/codewind-odo-extension) repository, create the ClusterRole with the required permissions, and bind that ClusterRole to the Che workspace service account.
+### Adding roles to support the extension
+The extension needs additional roles for accessing OpenShift resources. Use the following commands to clone the [codewind-odo-extension](https://github.com/eclipse/codewind-odo-extension) repository, create the ClusterRole with the required permissions, and bind that ClusterRole to the corresponding Codewind service account.
 
 ### Importing Java image stream to your OpenShift or OKD cluster
 In order to create or import Java compoent, you need to import Java image stream to your OpenShift or OKD cluster so that odo can build component image.
 
-### Adding rules and importing Java image stream
+### Adding roles and importing Java image stream
 1. Log in to your OpenShift or Origin Community Distribution (OKD) cluster.
-2. Enter the following commands to go to the correct location, add the rules and import Java image stream, and perform cleanup:
+2. Enter the following commands to go to the correct location, add the roles and import Java image stream, and perform cleanup:
```

Comment:
This is a minor thing, but should we add an article before "Java image stream"? For example in the title say, "importing the Java image stream" and in line 35, "and import the Java image stream".

Reply:
@sishida Yep, I already committed the changes, please review